src/elements/content-picker/Footer.js

```diff
@@ -58,21 +64,25 @@ const Footer = ({
         <div className="bcp-footer-right">
             {children}
 
-            <ButtonGroup className="bcp-footer-actions">
-                <Tooltip text={cancelButtonLabel || <FormattedMessage {...messages.cancel} />}>
-                    <Button onClick={onCancel} type="button">
-                        <IconClose height={16} width={16} />
-                    </Button>
-                </Tooltip>
-                <Tooltip
-                    isDisabled={!selectedCount}
-                    text={chooseButtonLabel || <FormattedMessage {...messages.choose} />}
-                >
-                    <PrimaryButton isDisabled={!selectedCount} onClick={onChoose} type="button">
-                        <IconCheck color="#fff" height={16} width={16} />
-                    </PrimaryButton>
-                </Tooltip>
-            </ButtonGroup>
+            {renderCustomActionButtons ? (
+                renderCustomActionButtons({ selectedCount, selectedItems })
```

Comment:
What do you think about passing down more props, in case the use case grows in the future? E.g. I'm wondering if onChoose or onCancel might still be useful props to pass down